src/blocks/sound.rs

```diff
@@ -653,6 +663,10 @@ impl SoundDevice for PulseAudioSoundDevice {
         self.active_port.clone()
     }
 
+    fn form_factor(&self) -> Option<String> {
+        self.form_factor.clone()
```

Comment:
This function can return `&str` in order to avoid `clone()`

Reply:
How's this?